### ISSUE TITLE
No server response

### ISSUE THREAD

**Issue by scottmeup** [simphax/eyetv-iptv-server#4]
Im not getting anything from the server when I connect to localhost:9898 in browser.
I've run `telnet localhost 9898 ` and the port is definitely open, but nothing is received and after a short time the remote host closes the port.
I've uninstalled the iptv server and confirmed with `sudo lsof -i -P -n | grep LISTEN | grep 9898` that nothing else is open on that port when the server has been removed.

I'm running 10.10.5, EyeTV 3.6 with iPhone access enabled.

**Comment by cjhdc** [comment]
Yes, replying to an old thread, but... fwiw, same issue here with 10.11 and EyeTV 3.6.9.
